mojaglobal-ui/README.md

```diff
@@ -0,0 +1,62 @@
+<div align="center">
```

Comment:
Is this file needed?

Reply:
Yes sir, this file is needed
The README.md file inside the mojaglobal-ui directory will appear on the npm package 
The README.md file in the root project will appear on the github repository